src/components/About/About.jsx

```diff
@@ -1,41 +1,44 @@
- import React from "react";
+import React from "react";
+import { renderBody as render } from "SHARED/renderUtils.jsx";
```

Comment:
we should probably export an object called `RENDER` from the shared renderUtils, with `renderBody` in there, so that we can still do things like `RENDER.body(data)`

Reply:
oop, OR, we could do

```
import * as RENDER from "SHARED/renderUtils.jsx";
```

although we'd have `RENDER.renderBody` instead of `RENDER.body` - I'll let you figure that fellah out!

Reply:
Using:
import RENDER from "SHARED/renderUtils.jsx"
&&
const body = RENDER.body(bodyData);

Quite close to Episodes approach, so seems good to me :)